kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,18 +1,22 @@
 return {
   fields = {
-    timeout = {type = "number", default = 60000, required = true },
-    keepalive = {type = "number", default = 60000, required = true },
-    aws_key = {type = "string", required = true},
-    aws_secret = {type = "string", required = true},
-    aws_region = {type = "string", required = true, enum = {
+    timeout = { type = "number", default = 60000, required = true },
```

Comment:
> It was explicit request from @thibaultcha. I would not ever change this because I agree that PR should be for single purpose.

No, please don't say such things. PR comments are always in the scope of said PR. I instructed you to update the code style of your change, and you went ahead and decided to update the entire table. That's undesired, but not my fault. Please propose the new line with the appropriate style, and leave the rest untouched.